chore(curriculum): validate skyline city project (#46708)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98cc.md

# --description--

> Nest these elements to `head` element, `meta` element with and attribute `charset` of `UTF-8`, the a `title` element, and give your project a title of `City Skyline`. Also, a self-closing `link` element, and give it a `rel` attribute value of `stylesheet` and an `href` attribute value of `styles.css`.

Reviewer: ```suggestion
Within the `head`, nest a `meta` element with a `charset` of `UTF-8`, a `title` element with a title of `City Skyline`, and a `link` element that links your `styles.css` file.
```

This seems quite a bit simpler - maybe too simple.

Author: the following project has a similar description 

![image](https://user-images.githubusercontent.com/88248797/176946348-ee35227d-1694-44a3-89f6-ef5076c07e11.png)
